Semantic Scholar.js

```diff
@@ -221,9 +133,7 @@ var testCases = [
 					}
 				],
 				"date": "2010",
-				"DOI": "10.1007/978-3-642-14770-8_33",
```

Comment:
We should figure out how to keep the DOI, one way or another, even if we have to get it via scraping rather than from the JSON-LD.

Reply:
Added the DOI back in by hitting the semantic scholar public api (which should have a very stable response format).